### PR TITLE
Fix broken links in howto-k8s-alb walkthrough

### PR DESCRIPTION
**Pull request by suniltheta**
**Issue**
https://github.com/aws/aws-app-mesh-examples/issues/474

**Description of changes:**
Fix some broken links on howto-k8s-alb walkthrough README.md.

New readme for ref: [README.md](https://github.com/suniltheta/aws-app-mesh-examples/blob/fix_howto-k8s-alb/walkthroughs/howto-k8s-alb/README.md)

On aws-load-balancer-controller [repo](https://github.com/kubernetes-sigs/aws-load-balancer-controller)
> This project was formerly known as "AWS ALB Ingress Controller", we rebranded it to be "AWS Load Balancer Controller"

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: walkthroughs/howto-k8s-alb/README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ## Overview
-This example shows how to use [ALB Ingress Controller](https://github.com/kubernetes-sigs/aws-alb-ingress-controller) with targets registered as virtual-nodes under App Mesh.
+This example shows how to use [AWS Load Balancer Controller](https://github.com/kubernetes-sigs/aws-load-balancer-controller) with targets registered as virtual-nodes under App Mesh.
 
 ![System Diagram](./howto-k8s-alb.png "System Diagram")
 
 ## Prerequisites
 - [Walkthrough: App Mesh with EKS](../eks/)
-- [Walkthrough: ALB Ingress Controller](https://kubernetes-sigs.github.io/aws-alb-ingress-controller/guide/walkthrough/echoserver/)
+- [Walkthrough: ALB Ingress Controller](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.3/examples/echo_server/)
```
Comment (cgchinmay) [inline walkthroughs/howto-k8s-alb/README.md:8]: we should update this name too

Comment (suniltheta) [reply walkthroughs/howto-k8s-alb/README.md]: fixed, thanks.

File: walkthroughs/howto-k8s-alb/README.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 ![System Diagram](./howto-k8s-alb.png "System Diagram")
 
 ## Prerequisites
 - [Walkthrough: App Mesh with EKS](../eks/)
-- [Walkthrough: ALB Ingress Controller](https://kubernetes-sigs.github.io/aws-alb-ingress-controller/guide/walkthrough/echoserver/)
+- [Walkthrough: AWS Load Balancer Controller](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.3/examples/echo_server/)
 - Install Docker. It is needed to build the demo application images.
 
-Note: Only [deploy the ALB Ingress Controller](https://kubernetes-sigs.github.io/aws-alb-ingress-controller/guide/walkthrough/echoserver/#deploy-the-alb-ingress-controller) and rest this example service will replace the echoserver in the ALB Ingress Controller link provider
+Note: Only [setup the AWS Load Balancer controller](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.3/examples/echo_server/#setup-the-aws-load-balancer-controller) and rest this example service will replace the echoserver in the ALB Ingress Controller link provider
```
Comment (cgchinmay) [inline walkthroughs/howto-k8s-alb/README.md:11]: Sorry for the delay but here as well ALB Ingress Controller -> AWS Load Balancer Controller

File: walkthroughs/howto-k8s-alb/README.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ You should see similar to the following.
       Normal  CREATE  11m    alb-ingress-controller  LoadBalancer 80113f18-howtok8salb-color-0f20 created, ARN: arn:aws:elasticloadbalancing:us-west-2:669977933099:loadbalancer/app/80113f18-howtok8salb-color-0f20/7c45f6fd4eefa871
```
Comment (cgchinmay) [inline walkthroughs/howto-k8s-alb/README.md:59]: we might need to the alb-ingress-controller, confirm it.

File: walkthroughs/howto-k8s-alb/README.md (hunk 2 above)
Comment (suniltheta) [reply walkthroughs/howto-k8s-alb/README.md]: fixed, this. Thanks for catching

File: walkthroughs/howto-k8s-alb/README.md (hunk 3 above)
Comment (suniltheta) [reply walkthroughs/howto-k8s-alb/README.md]: updated

**Review by cgchinmay — APPROVED** [review]
lgtm

**Review by flashyang — APPROVED** [review]
lgtm

**Pull request merged**